packages/context/src/context-connect.js

```diff
@@ -35,6 +35,12 @@ export function contextConnect(Component, namespace, options = {}) {
 		? namespace[0]
 		: namespace || WrappedComponent.name;
 
+	if (process.env.NODE_ENV === 'development') {
+		if (typeof namespace === 'undefined') {
+			console.warn('contextConnect', 'Please provide a namespace.');
+		}
+	}
+
```

Comment:
We can use `@wordpress/warning` for this 🙂

Reply:
We could, but I'd prefer not adding another `@wordpress/` dependency for now 🙈 haha